app/src/main/res/layout/layout_subscription_info.xml

```diff
@@ -39,56 +39,52 @@
       app:lottie_scale="1"
       />
 
-  <LinearLayout
+  <TextView
+      android:id="@+id/app_name"
+      style="@style/TextAppearance.AppCompat.Headline.Roboto.Medium"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
+      android:layout_gravity="start"
       android:layout_marginStart="12dp"
-      android:orientation="vertical"
-      app:layout_constraintBottom_toBottomOf="@id/app_icon"
+      android:ellipsize="end"
+      android:maxLines="1"
+      android:textColor="@color/black_87_alpha"
+      android:textSize="16sp"
```

Comment:
The text format when used is many places with the same configuration should be defined in a style.